### ISSUE TITLE
Two questions regarding authentication and locking

### ISSUE THREAD

**Issue by DumboJet** [WebDAVSharp/Example#3]
I see that the locking implementation is not on the latest nuget package.
Is it not release-ready yet?
Do you know of any cases where it fails?

Can you also demo how we can use authentication?
I use this code:
```
            var server = new WebDavServer(ref store, AuthType.Basic);
```
...but I don't see how we can handle the user authentication in code (e.g. validate the credentials against a DB, etc).
Maybe this is not yet implemented?

Thanks! :)

---

**UPDATE:**

Oh, I see now that when specifying basic authentication it uses Windows authentication by default!

```
        [DllImport("advapi32.dll", SetLastError = true, CharSet = CharSet.Unicode)]
        public static extern bool LogonUser(String lpszUsername, String lpszDomain, String lpszPassword, int dwLogonType, int dwLogonProvider, out SafeTokenHandle phToken);
```

Any plans to make this more flexible?

**Comment by EMerckx** [comment]
Dear DumboJet,

Currently, this project is on hiatus. 
Furthermore, I don't own the credentials to the NuGet package, @lievenjanssen does. My attempts in contacting him weren't successful.

The server should work fine with Windows Office 2010, but an issue was reported for Windows Office 2013. I also haven't been able to fully test the locking.

In our project, we only made use of Windows authentication.
But currently I don't have any plans to make it more flexible. There are different forks on the https://github.com/WebDAVSharp/WebDAVSharp.Server repository, maybe those can help you any further?

In short, currently there are no concrete plans, and I'm not able to update the NuGet package.
But feel free to contact me if you'd like some more information on the project.

Kind regards,
Ewout Merckx

**Comment by DumboJet** [comment]
Thank you for the valuable feedback @EMerckx ! :)
I will have a look at the forks and I will probably do some code changes myself. ;)
